src/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 ###############################################################################
-# Description       : CMake build script for libSBML
+# Description       : CMake build script for libLX
 # Original author(s): Frank Bergmann <fbergman@caltech.edu>
 # Organization      : California Institute of Technology
 #
-# This file is part of libSBML.  Please visit http://sbml.org for more
-# information about SBML, and the latest version of libSBML.
+# This file is part of libLX.  Please visit http://sbml.org for more
+# information about LX, and the latest version of libLX.
```

Comment:
I think `LX` should be `libLX` here

Reply:
the comments will need general revisions. sbml.org does not contain any information about liblx, and the documentation will reflect this later. I suggest leaving it till then.

Reply:
OK